go/service/ui_router.go

```diff
@@ -155,3 +155,20 @@ func (u *UIRouter) GetRekeyUI() (keybase1.RekeyUIInterface, int, error) {
 	}
 	return ret, sessionID, nil
 }
+
+func (u *UIRouter) GetRekeyUINoSessionID() (keybase1.RekeyUIInterface, error) {
```

Comment:
ok, so this is different from the normal case of CLI-versions of commands where the UID need not be delegated?